css/desktop.css

```diff
@@ -0,0 +1,361 @@
+:root {
+  --orange-color: #ec5242;
+  --dark-color: #272a31;
+  --gray-color: #d3d3d3;
+}
+
+@media only screen and (min-width: 768px) {
+  .desktop-hide {
+    display: none;
+  }
+
+  .section-layout {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 8rem 4rem 4rem 4rem;
+    gap: 1rem;
+  }
+
+  hr {
+    width: 3rem;
+    border: 1px solid var(--orange-color);
+  }
+
+  /* Navbar styling and background of the landpage */
+  #navbar {
+    position: fixed;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 4rem;
+    background-color: #fff;
+    border-bottom: 1px solid var(--gray-color);
+    gap: 2rem;
+  }
+
+  @media only screen and (max-width: 890px) {
+    #navbar {
+      padding: 0 1rem;
+    }
+  }
+
+  .category-card img {
+    width: 5rem;
+    height: 5rem;
+  }
+
+  .athlete-card img {
+    width: 10rem;
+    height: 10rem;
+  }
+
+  #navbar img {
+    width: 10rem;
+  }
+
+  #navbar ul {
+    display: flex;
+    gap: 2rem;
+  }
+
+  #navbar a {
+    text-decoration: none;
+    color: var(--dark-color);
+  }
+
+  #navbar a:hover {
+    color: var(--orange-color);
+  }
+
+  #navbar .tickets-link {
+    color: var(--orange-color);
+    text-transform: capitalize;
+    border: 5px solid var(--orange-color);
+    padding: 0.75rem;
+    transition: all 0.3s ease-in-out;
+  }
+
+  #navbar .tickets-link:hover {
+    color: #fff;
+    background-color: var(--orange-color);
+  }
```

Comment:
:red_circle:  It is considered a best practice to use the class selector over the ID selector as the class selector allows you to apply the same property on multiple elements. The main difference between an ID and a class is that an ID is only used to identify one single element in our HTML. IDs are only used when one element on the page should have a particular style applied to it. However, a class can be used to identify more than one HTML element. Kindly make sure this #navbar element includes a  .navbar class part of its class list   

`<nav id="navbar> class="navbar">`

then in your CSS, you can replace` #navbar` with `.navbar`